mungegithub/Makefile

```diff
@@ -53,6 +54,7 @@ endif
 # The cluster will need a github oauth token (the secret target makes that easy to create)
 deploy: push deployment
 	# Deploy the new deployment
+	kubectl --kubeconfig=$(KUBECONFIG) apply -f $(APP)/deployment/$(CONFIGMAP)/configmap.yaml
```

Comment:
Pushing a new configmap would have no effect on the running pod till it restarts. Is that okay? So, someone who does push a new configmap must restart the pod afterwards.